src/wallet/sqlite.cpp

```diff
@@ -219,6 +243,14 @@ void SQLiteDatabase::Open(const char* mode)
         if (ret != SQLITE_OK) {
             throw std::runtime_error(strprintf("SQLiteDatabase: Failed to enable fullfsync: %s\n", sqlite3_errstr(ret)));
         }
+
+        // Set the application id
+        uint32_t app_id = ReadBE32(Params().MessageStart());
+        std::string set_app_id = strprintf("PRAGMA application_id = %d", (int32_t)app_id);
```

Comment:
87e2d29baca9d08e90ed401bf360b9b7e866c62c
```suggestion
        std::string set_app_id = strprintf("PRAGMA application_id = %d", static_cast<int32_t>(app_id));
```

Reply:
Done